### DISCUSSION TITLE
Email Attachments 📦

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#3935] [Ideas]
As of today we cannot see attachments in messages threads.

The first step will be to display attachments within threads. 
The second step will be to bring attachments within the file tabs. 
The third step will be to enable adding attachments in message composer.

Syncing attachments locally would be very costly. Probably we should setup a passthrough/proxy that downloads the attachments on gmail using the user's connection token, and stream it back to our user.

**Comment by BOHEUS** [comment]
Related to https://github.com/twentyhq/core-team-issues/issues/225
